pkg/cli/prompt/list/list.go

```diff
@@ -134,6 +130,7 @@ func (m ListModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	switch msg := msg.(type) {
 	case tea.WindowSizeMsg:
 		m.width = msg.Width
+		m.List.SetSize(msg.Width, msg.Height-((3*msg.Height)/4))
```

Comment:
same here, let's add a comment for quick description why we are doing this since it might not be obvious from the code.